build.gradle

```diff
@@ -48,44 +48,46 @@ subprojects {
 
   /* PROJECT DEPENDENCY VERSIONS */
   // define all common versioned dependencies here
-  project.ext.dependencyVersions = [
+  project.ext.dependencyStrings = [
     // For Google libraries, check <http-client-bom.version>, <google.auth.version>, <guava.version>,
     // ... in https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/pom.xml
     // for best compatibility.
-    GOOGLE_HTTP_CLIENT: '1.34.0',
-    GOOGLE_HTTP_CLIENT_APACHE_V2: '1.34.0',
-    GOOGLE_AUTH_LIBRARY_OAUTH2_HTTP: '0.18.0',
-    GUAVA: '28.1-jre',
-    JSR305: '3.0.2', // transitively pulled in by GUAVA
+    GOOGLE_HTTP_CLIENT: "com.google.http-client:google-http-client:1.34.0",
```

Comment:
Really super-nit, but we try to maintain consistency regarding `'` and `"`. In Gradle (which is Groovy language), we use `'` for static strings like this one, while using `"` for strings containing expressions to evaluate (such as `"some text with a ${variable}"`).

Reply:
Thanks for pointing this out! Fixed the style issue.